packages/super-components/components/msc-billing-tile/src/msc-billing-tile/menu-custom.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
**question:** Why cant we have default export? I see only `MenuCustom` class is exported

Reply:
Because this generate a build error in stenciljs : there is no default export in these projects

https://stenciljs.com/docs/module-bundling#one-component-per-module
This is how we write file in stenciljs